.github/workflows/push.yml

```diff
@@ -21,6 +21,18 @@ jobs:
         run: |
           npm install
           npm run travis
+  build-docker-images:
+    needs: unit-tests
+    if: github.event_name == 'push'
```

Comment:
this line might not be required as I think it does the same thing as `on: push` on line 2, although it wouldn't hurt 😄